scripts/logistic_regression_clf.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import os
+
+import joblib
+from datasets import load_dataset
+from loguru import logger
+from sklearn.linear_model import LogisticRegression
+
+
+def main(args):
+    logger.info(f"Load dataset from {args.dataset_name}")
+    dataset = load_dataset(args.dataset_name)
+    dataset = dataset.class_encode_column("class")
+
+    train_ds = dataset["train"]
+    test_ds = dataset["test"]
+    logger.info(f"Train shape: {train_ds.shape}, Test shape: {test_ds.shape}")
+
+    logger.info(f"Train LR classifier for max {args.max_iter} iterations")
+    lr_clf = LogisticRegression(max_iter=args.max_iter)
```

Comment:
**Optionnaly**

make the clf using `GridSearch` of `Pipeline` of `StandardScaler` and `LogisticRegression` 

`Pipeline` and`StandardScaler` could be dropped